test/e2e/run_test.sh

```diff
@@ -226,6 +241,21 @@ while (( "$#" )); do
       SKIP_BUILD=true
       shift
     ;;
+    --use-cluster)
+      if [ -n "$2" ] && [[ "$2" != --* ]]; then
+        NAME="$2"
+        USE_EXISTING_CLUSTER=true
+        if ! kubectl config get-contexts | grep -q "$NAME"; then
```

Comment:
Using grep doesn't seem very rigorous. Why doesnt' it strictly match the result of `kind get clusters`